### ISSUE TITLE
Some filter setting not working?

### ISSUE THREAD

**Issue by asakurato** [erengy/taiga#508]
Soo, I'm trying to filter out batches and subtitle files. No matter how I try to discard and hide them, it still shows up in torrent page. I searched for the similar issues and found one, but neither rss solution, nor filter solution worked for me. Mainly, filtering out based on category doesn't seem to work (batches, raws, other). Hope this there is a solution for this.

**Comment by erengy** [comment]
As far as I'm aware, there's currently no solution other than what was previously discussed in #272 and #388. But I imagine that the "File size" element mentioned in #510 could be used for this purpose as well.

**Comment by asakurato** [comment]
The reason I had this question, because when the batches do appear (somehow, the ones that did appear when I posted this, no longer appear now), they are sorted in a section called "Bathes" So I thought they were already defined as batches and should be filtered by filters. Didn't know it was dependent on RSS. I tried RSS solution from 388, but it didn't work (probably because it was for old nyaa). File size element method looks good too, but I want to stick to nyaa for some reason. Guess I'll wait for file size filter that you also mentioned there.
